### ISSUE TITLE
Allow timestamps to be in UTC _or_ indicate the UTC offset

### ISSUE THREAD

**Issue by niconoe** [tdwg/camtrap-dp#81]
In GitLab by @peterdesmet on Jan 18, 2021, 21:25

Almost all timestamps in camtrap-dp require them to be expressed as UTC:

> as an ISO 8601 formatted string in UTC (e.g. `YYYY-MM-DDThh:mm:ssZ`)

I think it would be better if we **also** allowed ISO 8601 formatted string with UTC offsets. It can be easily converted to UTC, but carries a bit more information that is immediately accessible to the user:

```yaml
2020-01-18T13:00:00Z
2020-01-18T18:00:00-5:00 # Easier to have an immediate sense of the time of the day
```

So I propose to update definitions to:

> as an ISO 8601 formatted string with timezone designator (`YYYY-MM-DDThh:mm:ssZ` or `YYYY-MM-DDThh:mm:ss±hh:mm`)

Time fields:

- [x] ~~package.created~~ (data package property, best kept to UTC)
- [x] ~~package.temporal.start~~ (a date, does not apply)
- [x] ~~package.temporal.end~~ (a date, does not apply)
- [x] deployments.start
- [x] deployments.end
- [x] multimedia.timestamp
- [x] observations.timestamp
- [x] observations.classification_date

@kbubnicki ok?

**Comment by niconoe** [comment]
In GitLab by @peterdesmet on Jan 19, 2021, 10:16

assigned to @peterdesmet

**Comment by niconoe** [comment]
In GitLab by @kbubnicki on Jan 19, 2021, 12:44

Perfect!

**Comment by niconoe** [comment]
In GitLab by @peterdesmet on Jan 19, 2021, 18:20

mentioned in commit 111a226e97645446b38ef25f418dbd73b3bb1c64

**Comment by niconoe** [comment]
In GitLab by @peterdesmet on Jan 19, 2021, 18:21

Closed with https://gitlab.com/oscf/camtrap-dp/-/commit/111a226e97645446b38ef25f418dbd73b3bb1c64

**Comment by niconoe** [comment]
In GitLab by @jimcasaer on Jan 22, 2021, 20:44

@peterdesmet I suppose the aboven mentioned ones having TRUE idicate different camera time stamps ? being one hour later / earlier / equal to utc ?

**Comment by niconoe** [comment]
In GitLab by @peterdesmet on Jan 22, 2021, 21:17

TRUE indicates passing validation, with the column on the left different ways of writing timestamps (with the 4-5 clearly wrong, and 6 having no timezone).

**Comment by niconoe** [comment]
In GitLab by @kbubnicki on Jan 26, 2021, 13:17

Ok! Sounds reasonable to give this alternative option. I see no problem with this when this format also passes the validation.

**Comment by niconoe** [comment]
In GitLab by @peterdesmet on Jan 26, 2021, 13:39

mentioned in commit d7d0a6f99b79a466bc8cacabcecd1c37494d7414

**Comment by niconoe** [comment]
In GitLab by @peterdesmet on Jan 26, 2021, 13:57

Fixed in d7d0a6f99b79a466bc8cacabcecd1c37494d7414

**Comment by peterdesmet** [comment]
Fixed in #114
